setup.sh

```diff
@@ -5,18 +5,19 @@
 
```

Comment:
We can remove the comments with the two steps at the beginning here. The config step doesn't seem to be necessary anymore and the sun command is in the readme.